### ISSUE TITLE
Improve hide of elements and remove fade delay at initialization

### ISSUE THREAD

**Issue by prisky** [kartik-v/yii2-detail-view#13]
Noticed a strange behaviour - though explainable ... ish - without digging too deep. Thought I would mention in case it is a bug.

Wanted to set focus after the detail view drawn. The problem was focus was being called before the item was unhidden by the default 800 fadeDelay setting and the fact that kv-hide seems to be applied by default to input items when they are drawn. Setting fadeDelay to 0 had the strange behaviour on some of my views of leaving the inputs hidden. I guess this is result of show being called prior to the item being drawn due some kind of asynchronous mis-behaviour.

I am loading the detail view into a modal via ajax - not sure if this is having a bearing.

Basically, is there a reason for kv-hide to be applied when enableEditMode true in renderAttribute ?

For now as  fix I have overridden this and removed kv-hide from there, and all seems to be working ok now.

I would prefer not to override this if I can of course.

Cheers for the awesome extensions
